noir_by_example/simple_macros/noir/Nargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "simple_macros"
+type = "bin"
+authors = [""]
+compiler_version = ">=0.34.0"
```

The current default version of Noir is 0.33.
```suggestion
compiler_version = ">=0.33.0"
```

---

I'll update the readme to say `noirup -v 0.34.0` instead?
It's been in pre-release for a while now and should land soon.